kover-gradle-plugin/docs/aggregated.md

```diff
@@ -54,6 +54,44 @@ kover {
         
         // -Pkover.classes.includesAnnotated=*Included*
         includesAnnotatedBy.add("*Included*")
+
+        // -Pkover.classes.includesHeir=*ParentIncluded
+        includesInheritedFrom.add("*ParentIncluded")
+        
+        // -Pkover.classes.excludesHeir=*ParentExcluded
+        excludesInheritedFrom.add("*ParentExcluded")
+        
+        verify {
+            // -Pkover.verify.warn=true
+            warningInsteadOfFailure = true
+            
+            rule {
+                name = "custom name"
+                disabled = false
+                groupBy = GroupingEntityType.APPLICATION
+                
+                // specify filters for given rule, common filters will be inherited
+                // call `clearFilters()` to avoid common filters inheritance
+                filters {
+                    includedProjects.add(":a2")
+                    excludedProjects.add(":b2")
+                    includedClasses.add("classes.to.include2.*")
+                    excludedClasses.add("classes.to.exclude2.*")
+                    excludesAnnotatedBy.add("*.Generated2*")
+                    includesAnnotatedBy.add("*Included2*")
+                    includesInheritedFrom.add("*ParentIncluded2")
+                    excludesInheritedFrom.add("*ParentExcluded2")
+                }
+
+                bounds {
+                    // -Pkover.verify.min=1
```

Comment:
This is controlled by Gradle itself, if you specify `-Pkover.verify.min=1 -Pkover.verify.min=2`, then Gradle uses the most recent value: `providers.gradleProperty("kover.verify.min").get() == 2`.

Reply:
No, I mean in case:

```kotlin
verify {
  rule {
    bounds {
      minValue = 1
    }
  }
 
  rule {
    bounds {
      minValue = 2
    }
  }
}
```

if I do `./gradlew -Pkover.verify.min=5`, what is replaced by 5, `minValue = 1` or `minValue = 2`?

Reply:
`-Pkover.verify.min=...` works in append style.

If you specify 
```
verify {
  rule {
    bounds {
      minValue = 1
    }
  }
 
  rule {
    bounds {
      minValue = 2
    }
  }
}
```
and `./gradlew :test koverVerify -Pkover.verify.min=5` when 3 minimal bounds will be checked 1, 2 and 5

Reply:
Yes, this is not an obvious logic, however, as it is written in the documentation `-Pkover.verify.min=1` is equivalent of 
```
verify {
    rule {
        bounds {
            minValue = 1
         }
    }
}    
```

But if you add such code to the already written restrictions
```
verify {
    rule {
        bounds {
            minValue = 20
         }
    }
}

// added later
verify {
    rule {
        bounds {
            minValue = 1
         }
    }
}
```

then the rules merge - there are two minimal bounds: 20 and 1

Reply:
I think it should be explained in details somewhere